Mods/ArchipelagoPlayer.SC2Mod/enUS.SC2Data/LocalizedData/GameStrings.txt

```diff
@@ -2562,7 +2570,8 @@ Button/Tooltip/AP_PlanetaryFortressLoad=Loads nearby SCVs into the Planetary For
 Button/Tooltip/AP_PointDefenseDrone=Uses a laser to shoot down enemy missiles.  <n/>Cannot target special attacks. <n/>Times out after <d ref="Behavior,AP_PointDefenseDroneTimedLife,Duration"/> seconds.<n/><n/><c val="f078ff">Each shot consumes 10 energy.</c>
 Button/Tooltip/AP_PoisonNova=Instantly deals 20 damage to enemy ground units nearby.
 Button/Tooltip/AP_Predator=Anti-infantry specialist that deals area damage with each attack.<n/><n/><c val="#ColorAttackInfo">Can attack ground units.</c>
-Button/Tooltip/AP_PredatorFury=Jump from target to target, dealing <d ref="Effect,AP_DarkTemplarShadowFuryDamage,Amount"/> damage with each jump. Hits <d ref="Behavior,AP_PredatorShadowFuryController,PeriodCount+1"/> times.
+Button/Tooltip/AP_PredatorVespeneSynthesis=The player owning the Predator gains 1 Vespene every time Retribution Field deals damage.
+Button/Tooltip/AP_PredatorAdaptiveDefenses=Predators gain recharging shields, and take 50% reduced splash and ranged damage while the shield is active. A portion of damage dealt by Retribution Field also restores shields.<n/><n/>Shields have 2 base armor and are improved by Terran Vehicle Armor upgrades.

```

Comment:
Wording: The upgrade tooltips say `Retribution Field`, while the base trait says `Lightning Field`
![Snímek obrazovky pořízený 2024-08-31 18-07-02](https://github.com/user-attachments/assets/c41bf974-b384-42a8-87b9-e522c7564bde)
![Snímek obrazovky pořízený 2024-08-31 18-06-57](https://github.com/user-attachments/assets/e6e50aea-1402-48d2-92cf-66d73f0f186f)

Reply:
Fixed